src/index.js

```diff
@@ -3,7 +3,7 @@ const { SapphireClient, container } = require("@sapphire/framework");
 require("@sapphire/plugin-logger/register");
 
 // This is your custom store.
-const SlashCommandStore = require("./lib/stores/SlashCommandStore.js");
+const SlashCommandStore = require("./lib/structures/SlashCommandStore.js");
```

Comment:
As in my main comment, moved this to the same structure as @sapphiredev/framework